package.json

```diff
@@ -1,13 +1,17 @@
 {
   "name": "@dintero/email-chk",
-  "version": "0.0.0-development",
```

Comment:
revert the change... semantic-release will manage the version when creating new release